cmd/completion_test.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/astronomer/astro-cli/config"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCompletionRootCommand(t *testing.T) {
+	fs := afero.NewMemMapFs()
+	configYaml, err := ioutil.ReadFile("./testdata/config.yaml")
+	err = afero.WriteFile(fs, config.HomeConfigFile, []byte(configYaml), 0777)
+	config.InitConfig(fs)
+	expectedOut := `Error: accepts 1 arg(s), received 0
+Usage:
+  astro completion SHELL [flags]
+
+Flags:
+  -h, --help   help for completion
+
+`
+	_, output, err := executeCommandC(nil, fs, "completion")
+	assert.Equal(t, expectedOut, output, err)
```

Comment:
When called with not enough arguments like this the exit code on the shell is 1 - how do we test that here?

Reply:
it's fake execution, not sure that we can capture it here.